src/modules/filelist/FileOpener.jsx

```diff
@@ -26,103 +18,76 @@ export const getParentLink = element => {
 }
 
 const enableTouchEvents = ev => {
-  // remove event when you rename a file
+  // Check if the clicked element is an input to avoid disturbing events while renaming
   if (['INPUT', 'BUTTON'].indexOf(ev.target.nodeName) !== -1) {
     return false
   }
 
-  const parentDiv = getParentDiv(ev.target)
-  // remove event when it's the checkbox or the more button
-  if (
-    parentDiv.className.indexOf(styles['fil-content-file-select']) !== -1 ||
-    parentDiv.className.indexOf(styles['fil-content-file-action']) !== -1
```

Comment:
This condition is not necessary anymore. These buttons does not trigger the event at all.